app/Http/Controllers/DashboardController.php

```diff
@@ -302,15 +302,14 @@ public function getRecentInvoices()
         return Invoice::with("user:id,first_name,last_name,email,user_name")
             ->leftJoin("currencies", "invoices.currency", "=", "currencies.code")
             ->leftJoin("payments", "invoices.id", "=", "payments.invoice_id")
-            ->select("invoices.id as invoice_id", "invoices.number as invoice_number", "invoices.grand_total",
+            ->select("invoices.id as invoice_id", "invoices.number as invoice_number", "invoices.grand_total", "invoices.status as status",
```

Comment:
no need to write `as status`, since column name is status, it will come as status

Reply:
Fixed